Biz/Model/UserPasswordForm.pm

```diff
@@ -29,14 +29,14 @@ sub internal_initialize {
         require_context => 1,
         $self->field_decl(
             visible => [
-                [old_password => undef, 'NONE'],
-                qw(new_password confirm_new_password),
+                [qw(old_password Password NONE)],
+                [qw(new_password NewPassword NOT_NULL)],
+                [qw(confirm_new_password ConfirmPassword NOT_NULL)],
             ],
             hidden => [
-                [qw(display_old_password  Boolean)],
-                [query_password => undef, 'NONE'],
+                [qw(display_old_password Boolean)],
+                [qw(query_password String NONE)],
```

Comment:
I don't think we ever want to use String. It should always be limited, e.g. Text. It's a style thing here as it has no effect.